components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodClientProjectSessionTracker.kt

```diff
@@ -54,19 +55,26 @@ class GitpodClientProjectSessionTracker(
         }
     }
 
-    private fun isExposedServedPort(port: Status.PortsStatus?): Boolean {
+    private fun isExposedServedPort(port: PortsStatus?): Boolean {
         if (port === null) {
             return false
         }
         return port.served && port.hasExposed()
     }
 
+    private fun getForwardedPortUrl(port: PortsStatus): String {
+        return when {
+            manager.isAutoForwardingPorts -> "http://localhost:${port.localPort}"
```

Comment:
Is it really correct? Port forwarding can happen on another port if such local port is occupied we should read info from manager?

Reply:
True. When I wrote it, I thought JetBrains didn't have the functionality of forwarding to a different port. But I saw in the last meeting that they do have it.
I'll update it to use the port from JB Port Manager.

Reply:
Done. Now it's using the Client Port informed by JB Port Manager.